erpnext/healthcare/doctype/patient/patient.py

```diff
@@ -10,7 +10,7 @@
 from frappe.contacts.address_and_contact import load_address_and_contact
 from frappe.contacts.doctype.contact.contact import get_default_contact
 from frappe.model.document import Document
-from frappe.model.naming import set_name_by_naming_series
+from frappe.model.naming import set_name_by_naming_series,set_name_from_naming_options
```

Comment:
```suggestion
from frappe.model.naming import set_name_by_naming_series, set_name_from_naming_options
```


Please setup local linting to avoid reiterations:

https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist#linting

Reply:
Thanks, Ankush. I will do that.